[Add] 概要の英文を追加 (#23)
beaconFUN/mid-report2020: abstract.tex

%英語の概要
\begin{eabstract} Abstract in English. 

> The purpose of this project is install beacons in Hakodate real downtown, and develop services that utilize beacons. Through the service, create new value and provide people unique experiences and attractions.

Reviewer: ```suggestion
The purpose of this project is installing beacons in Hakodate real downtown, and develop services that utilize beacons. Through the service, create new value and provide people unique experiences and attractions.
```

Hakodate City has lots of attractions such as "food," "history," and "scenery."
In terms of food, Hakodate City faces the sea, so you can eat fresh and delicious seafood.

> In addition, there are many historical buildings because many historical events was happened in Hakodate city.

Reviewer: ```suggestion
In addition, there are many historical buildings because many historical events were happened in Hakodate city.
```

> In addition, there are many historical buildings because many historical events was happened in Hakodate city.

Reviewer 2: ```suggestion
In addition, there are many historical buildings because many historical events was happened in Hakodate City.
```
cityとCityの表記ゆれ？

> In addition, there are many historical buildings because many historical events was happened in Hakodate City.

Author: ```suggestion
In addition, there are many historical buildings because many historical events were happened in Hakodate City.
```

> In addition, there are many historical buildings because many historical events were happened in Hakodate City.

Author: ```suggestion
In addition, there are many historical buildings because many historical events happened in Hakodate City.
```

Furthermore, "Night view of Hakodate" is one of the top 3 night views in Japan.
Due to the existence of many such attractive tourism resources, many tourists visit every year.
The total number of tourists was about 5,369,000 in 2019.

> From this number, it can be seen that the tourism industry has become an important basic industry for Hakodate City.

Reviewer 3: 基幹産業のことを
英語で"Key Industry"っていうぞ！
![寿司ゆき:smile](https://d1zd1v0cxnbx2w.cloudfront.net/images/sets/sushiyuki/25.png) 
```suggestion
From this number, it can be seen that the tourism industry has become an important key industry for Hakodate City.
```

Reviewer 3: 鍵産業 うっ頭が...
![key](https://user-images.githubusercontent.com/40351476/93611887-ce3d6080-fa09-11ea-8e15-bed987a960de.png)

> Therefore, it is necessary to continue to tell the attractions from various angles so that Hakodate city make people think they wants to visit this attractions city many times.

Reviewer 2: ```suggestion
Therefore, it is necessary to continue to tell the attractions from various angles so that Hakodate City make people think they wants to visit this attractions city many times.
```
ここもcityとCity

> These days, IoT has become widespread due to improvements in communication technology, miniaturization and cost reduction of communication equipment.

Author: ```suggestion
These days, IoT has become widespread due to improvements in communication technology, and miniaturization and cost reduction of communication equipment.
```

We would like to use beacons to bring the attractions to more people in new ways.
In order to achieve that purpose, we devised our activities.

> First, we use a original task management method based on agile development methods.

Reviewer 4: ```suggestion
First, we use an original task management method based on agile development methods.
```

By introducing the backlog, managing tasks became easier.

> In addition, using task management method makes it is easy to introduce agile development methods when we move to the development phase.

Author: ```suggestion
In addition, using the task management method makes it is easy to introduce agile development methods when we move to the development phase.
```

> The second is that we didn't decide a project leader.

Author: ```suggestion
The second is that we didn't decide on a project leader.
```

> As a result, an environment is realized in which all members have responsibility and are easy to work.

Author: ```suggestion
As a result, the environment is realized in which all members have a responsibility and we are easy to work.
```

> We did fieldwork for proposing services, and identified issues and demands hidden in Hakodate city.

Reviewer: ```suggestion
We did fieldwork for proposing services and identified issues and demands hidden in Hakodate city.
```

> We did fieldwork for proposing services, and identified issues and demands hidden in Hakodate city.

Reviewer 2: ```suggestion
We did fieldwork for proposing services, and identified issues and demands hidden in Hakodate city.
```
city,City

> In addition, we created some ideas from the perspectives of "how typical of hakdoate," "why we use beacon," "necessity of the service," "novelty of the service," and "unchangeable attractive (continuity) of the service."

Reviewer: ```suggestion
In addition, we created some ideas from the perspectives of "how typical of hakodate," "why we use beacon," "necessity of the service," "novelty of the service," and "unchangeable attractive (continuity) of the service."
```

> In addition, we created some ideas from the perspectives of "how typical of hakodate," "why we use beacon," "necessity of the service," "novelty of the service," and "unchangeable attractive (continuity) of the service."

Author: ```suggestion
In addition, we created some ideas from the perspectives of "how typical of Hakodate City," "why we use beacon," "necessity of the service," "novelty of the service," and "unchangeable attractive (continuity) of the service."
```

> We narrowed it down nine ideas using that view points.

Reviewer 3: view points
って分けるんやっけ？　今まで
viewpoints
 ってつなげて書いてた

> We narrowed it down nine ideas using that viewpoints.

Author: ```suggestion
We narrowed it down 9 ideas using that viewpoints.
```

> In the second semester, we will brush-up and further refine these ideas to 3 or 4 ideas.

Author: ```suggestion
In the second semester, we will brush-up and further refine these ideas into 3 or 4 ideas.
```

> In the second semester, we will brush-up and further refine these ideas to three or four ideas.

Author: ```suggestion
In the second semester, we will brush-up and further refine these ideas to 3 or 4 ideas.
```

> After that, we are going to make statement and requirement definition of the services for developing.

Author: ```suggestion
After that, we are going to make a statement and requirement definition of the services for developing.
```

> After that, we are going to make statement and  requirement definition of the services for developing.

Reviewer: ```suggestion
After that, we are going to make statement and requirement definition of the services for developing.
```
